### PR TITLE
CivilWar in Java

### PR DESCRIPTION
**Pull request by poblish**
Happy Path seems to be fine: I can match the output at https://www.atariarchives.org/basicgames/showpage.php?page=46 with just some small rounding errors.

Needs some work to bring the object orientation up to spec, restore input validation, cover some more of the (many) edge cases, tweak rounding and column alignment etc.

### CODE REVIEW

File: 27_Civil_War/civilwar.bas [hunk 1]
```diff
@@ -216,7 +216,7 @@
 2280  PRINT "CASUALTIES",C5,C6
 2290  PRINT "DESERTIONS",INT(E),INT(E2)
 2300  PRINT
-2310  IF B$ <> "YES" THEN 2350
+2310  IF B$ <> "YES" THEN 2530
```
Comment (poblish) [inline 27_Civil_War/civilwar.bas:219]: 2530 is correct according to the listing at https://www.atariarchives.org/basicgames/showpage.php?page=49

**Comment by coding-horror** [comment]
Excellent, thanks for this, will merge when ready! 💪

**Comment by akieu222** [comment]
Good

**Comment by coding-horror** [comment]
LMK when we can merge this!

**Comment by coding-horror** [comment]
Excellent!! Thank you! 👏

**Pull request merged**